cmd/guacone/cmd/files.go

```diff
@@ -212,10 +212,14 @@ func getProcessor(ctx context.Context) (func(*processor.Document) (processor.Doc
 }
 func getIngestor(ctx context.Context) (func(processor.DocumentTree) ([]assembler.Graph, error), error) {
 	return func(doc processor.DocumentTree) ([]assembler.Graph, error) {
-		inputs, err := parser.ParseDocumentTree(ctx, doc)
+		inputs, idStrings, err := parser.ParseDocumentTree(ctx, doc)
```

Comment:
is this needed? If not used can remove `idStrings` and move the below comment here.